### PR TITLE
Prepare release 0.3

### PR DESCRIPTION
**Pull request by maxnoe**
@lmohrmann I would be very grateful if you could build the docs locally and have a look through if it looks okay...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ Stable versions of the spec are done via git tags and are shown as releases here
...
 HTML and PDF versions for stable versions are available via the version selector
-in the lower left on ReadTheDocs. As an example, for version 0.2:
+in the lower left on ReadTheDocs. As an example, for version 0.3:
 
-- HTML at <https://gamma-astro-data-formats.readthedocs.io/en/v0.2/>
-- PDF at <https://media.readthedocs.org/pdf/gamma-astro-data-formats/v0.2/gamma-astro-data-formats.pdf>
+- HTML at <https://gamma-astro-data-formats.readthedocs.io/en/v0.3/>
+- PDF at <https://media.readthedocs.org/pdf/gamma-astro-data-formats/v0.3/gamma-astro-data-formats.pdf>
 
 For v0.2 we also archived the sources of the spec as well as a rendered PDF and HTML version here:
```
Comment (maxnoe) [inline README.md:20]: The doi can only be updated after we made the release, right? Or was it uploaded manually? In that case we could pre-register a DOI with zenodo and already include it here.

@lmohrmann @cdeil ?

Comment (adonath) [reply README.md:20]: Can't we just always link to the latest release here? Or put a badge instead, the link would be https://zenodo.org/record/1409830

Comment (adonath) [reply README.md:20]: Ok, I just realize the purpose of the entry in the README is different...

**Comment by lmohrmann** [comment]
I'm afraid I won't get to build the docs locally in the next 2-3 weeks or so, sorry. But you shouldn't wait for me - I'm sure things look OK, and if they don't we fix them and release 0.4 😇

**Comment by maxnoe** [comment]
Ok, so from what I can tell, the upload to zenodo has been done manually.

I reservered a doi now: `10.5281/zenodo.7236105` which we can use to upload. I will add it to the README.

**Comment by maxnoe** [comment]
@adonath Merge and tag?

File: README.md [hunk 2]
```diff
@@ -12,14 +12,19 @@ Stable versions of the spec are done via git tags and are shown as releases here
...
-For v0.2 we also archived the sources of the spec as well as a rendered PDF and HTML version here:
-<https://doi.org/10.5281/zenodo.1409830> 
-To cite that version, you can use the bibtex entry [here](https://zenodo.org/record/1409831/export/hx#.W5EBLNgzY_U).
+
+The archived versions of the standard are available on zenodo:
+- v0.3:
+  - <https://doi.org/10.5281/zenodo.7236105>
+  - [BibTeX citation](https://zenodo.org/record/7236105/export/hx#.W5EBLNgzY_U).
+- v0.2:
+  - <https://doi.org/10.5281/zenodo.1409830>
```
Comment (adonath) [inline README.md:26]: I think here is one more error: `zenodo.1409830` will resolve to the latest verion. So put the v0.2 here instead:

```suggestion
  - <https://doi.org/10.5281/zenodo.1409831>
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Data formats for gamma-ray astronomy [![CC-BY 4.0](https://licensebuttons.net/l/by/4.0/88x31.png)](https://creativecommons.org/licenses/by/4.0/) [![Build Status](https://travis-ci.com/open-gamma-ray-astro/gamma-astro-data-formats.svg?branch=master)](https://travis-ci.com/open-gamma-ray-astro/gamma-astro-data-formats) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1409831.svg)](https://doi.org/10.5281/zenodo.1409831)
+# Data formats for gamma-ray astronomy [![CC-BY 4.0](https://licensebuttons.net/l/by/4.0/88x31.png)](https://creativecommons.org/licenses/by/4.0/) [![Build Status](https://travis-ci.com/open-gamma-ray-astro/gamma-astro-data-formats.svg?branch=master)](https://travis-ci.com/open-gamma-ray-astro/gamma-astro-data-formats) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7236105.svg)](https://doi.org/10.5281/zenodo.7236105)
```
Comment (adonath) [inline README.md:1]: I think for the badge we should just always resolve to the latest version like:
```suggestion
# Data formats for gamma-ray astronomy [![CC-BY 4.0](https://licensebuttons.net/l/by/4.0/88x31.png)](https://creativecommons.org/licenses/by/4.0/) [![Build Status](https://travis-ci.com/open-gamma-ray-astro/gamma-astro-data-formats.svg?branch=master)](https://travis-ci.com/open-gamma-ray-astro/gamma-astro-data-formats) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1409830.svg)](https://doi.org/10.5281/zenodo.1409830)
```
This prevents the future need to update manually...

**Review by adonath — CHANGES_REQUESTED** [review]
Thanks @maxnoe, I have left two more inline suggestions...

File: README.md (hunk 3 above)
Comment (maxnoe) [reply README.md]: fixed

**Review by adonath — APPROVED** [review]
Thanks @maxnoe, no further comments from my side!

**Comment by maxnoe** [comment]
Ok, let's merge and make the release then, thanks!

Note: we need to contact @cdeil for the zenodo release, but we can do the tag and upload to readthedocs already

**Pull request merged**